AwsCryptographyPrimitives/runtimes/java/src/main/java/ECDH/ECCUtils.java

```diff
@@ -378,4 +406,109 @@ public static java.security.spec.ECParameterSpec ecParameterSpec(
       java.security.spec.ECParameterSpec.class
     );
   }
+
+  public static Result<
```

Comment:
nit: if this is mostly for testing, and always fails, we should probably include a javadoc comment saying so, so that consumers don't mistake this for a useful utility function...it also could maybe be moved into a separate class, but i don't feel so strongly about it.

Reply:
ack; I can add a small comment. This extern is not exposed in the primitives client, so if someone were to use this they are breaking the primitive client contract anyways...